.github/workflows/sv_pipeline_docker.yml

```diff
@@ -205,7 +132,7 @@ jobs:
       - name: Run build_docker.py
         run: |
           cd ./scripts/docker/
-          python build_docker.py --targets ${{ needs.build_args_job.outputs.target_images }} --image-tag ${{ needs.build_args_job.outputs.images_tag }}
+          python build_docker.py --base-git-commit ${{ needs.build_args_job.outputs.base_sha }} --current-git-commit ${{ needs.build_args_job.outputs.head_sha }} --image-tag ${{ needs.build_args_job.outputs.images_tag }}
```

Comment:
There's no need to compute and pass --current-git-commit. In this context, the correct value is always the current git repo status, which is the default.

Reply:
To me, the situation is that the calculation of the head commit for a non-PR is complicated and involves piping stuff through jq, sed, etc. As you say, the final commit, regardless of being a surrogate must have the same changes to project files as the head commit that was pushed, otherwise the merge couldn't work. So I just see another opportunity to strip complexity out of this script. And I'm waiting for the day when github changes its API slightly, at which point we'd only have one thing to figure out how to compute (the base commit). Since the PR as written works, I'm not going to insist.

Reply:
Thank you!